### PR TITLE
feat:build home page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Screenshot (117)  Screenshot (118)

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  border: 1px solid transparent;
+  border-radius: 10px;
+  padding: 4rem;
+  color: var(--white);
+  font-weight: 600;
+  font-size: 1.5rem;
+  width: 15rem;
+  height: 10rem;
+  text-align: center;
+  box-shadow: 0 3px 10px rgba(211, 209, 209, 0.2);
```
Comment (Naveenchandar) [inline index.css]: Please use hex code for consistency.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+}
+
+.footer {
+  background-color: var(--primary-color);
+  color: var(--white);
+  padding: 1rem;
+}
+
+.footer-items {
+  display: flex;
```
Comment (Naveenchandar) [inline index.css]: If flex property is repeated often, either you can declare a variable or separate class name in util file.

Comment (ishitakeshawani) [reply index.css]: done

File: index.css (hunk 1 above)
Comment (ishitakeshawani) [reply index.css]: yes

**Review by Naveenchandar — APPROVED** [review]
Looks good.

File: index.css [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  color: var(--bg-color);
+}
+
+.footer-social-icon {
+  color: var(--white);
+  font-size: 1.5rem;
+  margin: 0 0.5rem;
+}
+
+@media only screen and (max-width: 43rem) {
```
Comment (Aditi0505) [inline index.css]: We preferably use em or px with media queries. You can convert this.

**Review by Aditi0505 — APPROVED** [review]
Looks good to me.

**Review by aurobindoGupta — APPROVED** [review]
noice

File: index.css (hunk 3 above)
Comment (ishitakeshawani) [reply index.css]: yes, Thanks for reminding.

**Pull request merged**